addons/udes_stock_picking_batch/tests/test_stock_picking_batch.py

```diff
@@ -1046,69 +1044,62 @@ def test_get_batches_assigned_to_a_user(self):
 
         self.assertEqual(batch, searched_batch)
 
-    def test_assign_batches_for_batch_with_multiple_picking_types(self):
+    def test_batch_picking_type_is_assigned_when_batch_is_assigned_during_pick_creation(self):
         """
-        Create two pickings, assign the pickings to the batch, put the batch in a ready state and then request that the
-        one of the pickings
+        Check that the batch picking type is assigned when a picking is created and assigned a batch
+        at the same time.
         """
-        self.create_quant(self.apple.id, self.test_received_location_01.id, 4)
         self.create_quant(self.banana.id, self.test_stock_location_01.id, 4)
 
-        batch = self.create_batch(user=self.outbound_user)
+        batch_pick = self.create_batch(user=False)
```

Comment:
Any reason for changing this name?  It seems sensible to name the thing that `create_batch` returns "batch".

Reply:
We don't want to assign a user to the batch. create_batch() will assign cls.env.user if user parameter is None.

Reply:
I mean changing the name from `batch` to `batch_pick`...

Reply:
Oh sorry I completely misread it. No, I was copying code around and I wrote it with a more specific name because I used it somewhere else. Not any particular reason.

Reply:
I think it would a little better in reverse - `pick_batch` and `putaway_batch` - I think that makes it clearer the variable refer to batches.

Reply:
Addressed, sorry for force pushing (bad habit), at least it's a minor change.